app/drivers/zephyr/dts/bindings/sensor/alps,ec11.yaml

```diff
@@ -19,3 +19,7 @@ properties:
     type: int
     description: Number of pulses per tick
     required: false
```

Comment:
Is there a way we can mark this as deprecated?

Reply:
Looks like it: https://docs.zephyrproject.org/3.2.0/build/dts/bindings.html#property-entry-syntax

Will do.

Reply:
Added.